lib/systems/include/CASESystem.h

```diff
@@ -148,10 +148,17 @@ class CASESystem
 
     DrivetrainCommand_s get_current_drive_command() { return current_command_; }
 
+    /// @brief uses pedal data to determine the requested torque by the car
```

Comment:
What does requested torque by "the car" mean? By the drive brain? By this particular controller? What is "the car"?